mod-orders-storage/examples/piece_audit_event.sample

```diff
@@ -15,7 +15,9 @@
       "locationId": "53cf956f-c1df-410b-8bea-27f712cca7c0",
       "poLineId": "d471d766-8dbb-4609-999a-02681dea6c22",
       "titleId": "9a665b22-9fe5-4c95-b4ee-837a5433c95d",
+      "receivingTenantId": "d572c162-7ce1-5c15-d7ac-234c149a125b",
```

Comment:
it's should be word instead of uuid, so you can use 'diku' in samples instead

Reply:
Modified all occurrences